src/mpcCoreKit.ts

```diff
@@ -280,12 +280,14 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
 
     this.ready = true;
 
-    // setup session Manager during init instead of async constructor
-    const sessionId = await this.currentStorage.get<string>("sessionId");
-    this.sessionManager = new SessionManager({
-      sessionTime: this.options.sessionTime,
-      sessionId,
-    });
+    if (this.options.sessionTime !== 0) {
```

Comment:
in constructor sessionTime is being set to 86400 if nothing or 0 is passed, so i think this wont work until we handle that